woocommerce/woocommerce-ios
# [IPPInAppFeedback] Update and save feedback status upon merchant submission

Closes: #8729 


## Description
This PR updates the IPP survey feedback status when a merchant taps in the "Share Feedback" button, and marks it as submitted by setting its `FeedbackSettings.Status` to `.given(Date())`

## Testing instructions
On a store with COD enabled, and US/CA:

1. Go to Orders > See the IPP survey banner > Tap on "Share Feedback" > See the feedback being dismissed
2. Re-run the simulator or close/re-open the app
3. See how the IPP survey banner feedback does not appear.
